components/Navigation.js

```diff
@@ -0,0 +1,55 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+const Navigation = () => {
+  const router = useRouter();
+
+  return (
+    <NavBar>
+      <NavItem $active={router.pathname === "/"}>
+        <Link href="/">I</Link>
+      </NavItem>
```

Comment:
Having this HTML structure will make it so that clicking the Link is really difficult. 

I suggest removing the wrapping `div` and styling the Link to look the way you wish.

```suggestion
      <StyledLink href="/" $active={router.pathname === "/"}>
        I
      </StyledLink>
```